kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/client/ServiceDiscovery.scala

```diff
@@ -342,6 +343,31 @@ object ServiceDiscovery extends Logging {
     }
     serviceNode
   }
+
+  def withLock(
+      zkClient: CuratorFramework,
+      lockPath: String,
+      lockTimeOutMilliseconds: Long)(f: => Unit): Unit = {
```

Comment:
Rename `lockTimeOutMilliseconds` => `lockTimeout`.
Personally, I agree with that any time variable should indicate the unit of values, but kyuubi already choose another style, all time should represent by `Long` in milliseconds, let's keep same style.

Reply:
Update